Source/Scene/Scene.js

```diff
@@ -2556,6 +2556,10 @@ define([
         var frameNumber = CesiumMath.incrementWrap(frameState.frameNumber, 15000000.0, 1.0);
         updateFrameState(scene, frameNumber, time);
         frameState.passes.render = true;
+
+        var backgroundColor = defaultValue(scene.backgroundColor, Color.BLACK);
```

Comment:
Instead of duplicating this `defaultValue` with the same call elsewhere in the render loop, perhaps we want a property get/set so that we can just throw a `DeveloperError` on set when this is undefined?